src/common/offsetsJP.asm

```diff
@@ -0,0 +1,181 @@
+; ----------------------------------------------------------------------
+; Copyright © 2023 End45
```

Comment:
Technically not correct since I didn't write the file, but I don't know the details of how this should be specified. Do I need to update all the files? Just this one? Does it even matter?

Reply:
I guess I could change it to just Adex? I’m not super well-versed in copyright shenanigans but it makes sense for it to change, even if it’s very derivative of the other offset files.

Reply:
> I don’t see any changes to when Irdkwia updated moveShortcuts

Yeah, mostly because I also didn't know if it was necessary lol

I think the comment is only informative, whatever the correct copyright distribution is, it's not going to change because a comment says so.
Maybe one day I'll just update them all to say "End45 and contributors". But I guess this should be ok for now (if you don't mind, of course).

Reply:
Leaving it as End45 for consistency